inspirehep/dojson/hep/fields/bd6xx.py

```diff
@@ -106,37 +77,125 @@ def accelerator_experiments2marc(self, key, value):
     }
 
 
-@hep.over('thesaurus_terms', '^695..')
-def thesaurus_terms(self, key, value):
-    """Controlled keywords."""
-    value = force_force_list(value)
+@hep.over('keywords', '^695..')
+@hep.over('keywords', '^653[10_2][_1032546]')
+@utils.for_each_value
+def keywords(self, key, value):
+    """Parse keywords.
+
+    We have 2 types of keywords from marc, the 'thesaurus_terms' and
+    'freekeys', and we want to merge them for json into a big 'keywords' and
+    extract the 'energy_ranges' too that is represented in marc as special
+    value of some keywords.
+    """
+    def _is_thesaurus(value):
+        return '9' not in value
```

Comment:
Ahh, I see what you mean, you mean the '695' key that was matched against. Agree, will change.